src/main/java/com/greedobank/cards/model/Card.java

```diff
@@ -0,0 +1,69 @@
+package com.greedobank.cards.model;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.greedobank.cards.utils.CurrencyType;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import java.time.OffsetDateTime;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Card {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "card_id", updatable = false, nullable = false)
+    private int id;
+
+    @Column(name = "first_name")
+    private String firstName;
+
+    @Column(name = "last_name")
+    private String lastName;
+
+    @Column(name = "number")
+    private String number;
+
+    @Column(name = "pin")
+    private String pin;
+
+    @Column(name = "cvv")
+    private String cvv;
+
+    @Column(name = "end_date")
+    private OffsetDateTime endDate;
+
+    @Column(name = "active")
+    private boolean active;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "currency_type")
+    private CurrencyType currency;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "card_template_id")
+    @JsonBackReference
```

Comment:
what's the purpose of JSON-related annotations here? what if we switch to/add XML or smth else?